mesonbuild/mesonlib/vsenv.py

```diff
@@ -66,7 +67,10 @@ def _setup_vsenv(force: bool) -> bool:
         # VS installer instelled but not VS itself maybe?
         raise MesonException(f'Could not parse vswhere.exe output')
     bat_root = pathlib.Path(bat_info[0]['installationPath'])
-    bat_path = bat_root / 'VC/Auxiliary/Build/vcvars64.bat'
+    if platform.machine() == 'ARM64':
+        bat_path = bat_root / 'VC/Auxiliary/Build/vcvarsx86_arm64.bat'
+    else:
+        bat_path = bat_root / 'VC/Auxiliary/Build/vcvars64.bat'
     if not bat_path.exists():
         raise MesonException(f'Could not find {bat_path}')
 
```

Comment:
Proposed fix: https://github.com/mesonbuild/meson/pull/9413

Reply:
Technically a PR is a ticket with a patch attached to it. :D

Anyway it's not all that much work and I do understand the confusion since it's common in corporate OSS at least, and meson doesn't have a contributor guidelines document specifically stating whether or not to bother.

Reply:
@gaborkertesz-linaro FYI, when a PR fixes an issue, you should add `Fixes: #1234` in the commit message, that will tell GitHub to automatically close the issue once the PR is merged.